pkg/egressgateway/manager.go

```diff
@@ -45,6 +54,20 @@ const (
 	eventDeleteEndpoint
 )
 
+type Config struct {
+	// Install egress gateway IP rules and routes in order to properly steer
+	// egress gateway traffic to the correct ENI interface
+	InstallRoutes bool
+}
+
+var DefaultConfig = Config{
```

Comment:
Does this have to be exported? Would prefer it to be priv so it's not possible to mutate it from outside

Reply:
It's exported because of the ugly "enable egw" hack, I'll try to get rid of it.